frontend/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx

```diff
@@ -68,14 +68,19 @@ export class UploadModelFileDialog extends React.PureComponent<IProps, IState> {
 	}
 
 	public handleFileUpload = (values) => {
-		const { modelId, teamspaceName, projectName, handleClose, uploadModelFile } = this.props;
+		const { modelId, teamspaceName, projectName, handleClose, uploadModelFile, modelName } = this.props;
 
 		const fileData = {
 			file: values.file,
 			tag: values.revisionName,
 			desc: values.revisionDesc
 		};
-		uploadModelFile(teamspaceName, projectName, modelId, fileData);
+
+		const modelData = {
+			modelName, modelId
+		};
+
+		uploadModelFile(teamspaceName, projectName, modelData, fileData);
```

Comment:
I think this breaks the functionality. 

The uploadModelFile  function  that gets called is actually here:
https://github.com/3drepo/3drepo.io/blob/4743a3927cbf4cce777104739fae2be9e3f6f423/frontend/modules/model/model.sagas.ts#L101

And it expects the third parameter to be just the modelId

Reply:
Resolved, as error when copying and not merging correctly.